pom.xml

```diff
@@ -231,7 +235,17 @@
             <dependency>
                 <groupId>org.slf4j</groupId>
                 <artifactId>slf4j-api</artifactId>
-                <version>1.7.10</version>
+                <version>1.7.21</version>
```

Comment:
Suggest using a property for this as suggested at https://wiki.jasig.org/display/PLT/Logging+Best+Practices.  Also FWIW I prefer all the logging dependencies to be packaged together in one area with a comment.  That's the way it was done with the other portlets to make it clear.  For instance I see log4j and commons-logging exclusions being removed, but without the 'provided' scope you'll end up getting conflicting logging jars in the WEB-INF/lib folder causing trouble.